src/components/error-pane/index.css

```diff
@@ -1,12 +1,12 @@
-.error-pane .error {
+.error-pane__list-item--error {
```

Comment:
I'm not a fan of long CSS names. If you have enough time, maybe it's worth rewriting everything to be more semantic with something like tachyons.

Reply:
Yes, BEM class names tend to be quite long.
So should i avoid using BEM and use tachyons classes instead?

Reply:
I've actually never used BEM and I can't say I'm convinced by its value yet. Let me close this pull request since I don't want to adopt it now.